Programmers/230915_구명보트/kwaksj329.js

```diff
@@ -0,0 +1,19 @@
+function solution(people, limit) {
+  let answer = 0;
+  people.sort((a, b) => a - b);
+  let heavy = people.length - 1;
+  let light = 0;
+
+  while (heavy >= light) {
+    if (heavy !== light && people[heavy] + people[light] <= limit) {
+      answer++;
+      heavy--;
+      light++;
+    } else {
+      answer++;
+      heavy--;
+    }
```

Comment:
```suggestion
    if (heavy !== light && people[heavy] + people[light] <= limit) {
      light++;
    }
    heavy--;
    answer++;
```
이렇게 하면 중복 코드를 살짝 줄일 수 있겠네요!